COMETwebapp/ViewModels/Pages/UserManagement/UserManagementPageViewModel.cs

```diff
@@ -0,0 +1,165 @@
+    
+    using COMETwebapp.Pages.UserManagement;
+    using COMETwebapp.SessionManagement;
+    
+    using DevExpress.Blazor;
+    
+    using DynamicData;
+    
+    using ReactiveUI;
+
+    /// <summary>
+    ///     View model for the <see cref="UserManagementPage" /> page
+    /// </summary>
+    public class UserManagementPageViewModel : ReactiveObject, IUserManagementPageViewModel
+    {
+        /// <summary>
+        ///     The <see cref="Person" /> to create
+        /// </summary>
+        public Person Person { get; set; } = new();
+
+        /// <summary>
+        ///     The <see cref="EmailAddress" /> to create
+        /// </summary>
+        public EmailAddress EmailAddress { get; set; } = new();
+
+        /// <summary>
+        ///     The <see cref="TelephoneNumber" /> to create
+        /// </summary>
+        public TelephoneNumber TelephoneNumber { get; set; } = new();
+
+        /// <summary>
+        ///     Gets or sets the data source for the grid control.
+        /// </summary>
+        public SourceList<Person> DataSource { get; } = new();
+
+        /// <summary>
+        ///    Available <see cref="Organization"/>s
+        /// </summary>
+        public IEnumerable<Organization> AvailableOrganizations { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="PersonRole"/>s
+        /// </summary>
+        public IEnumerable<PersonRole> AvailablePersonRoles { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="DomainOfExpertise"/>s
+        /// </summary>
+        public IEnumerable<DomainOfExpertise> AvailableDomains { get; set; }
+
+        /// <summary>
+        ///    Available <see cref="VcardEmailAddressKind"/>s
+        /// </summary>
+        public IEnumerable<VcardEmailAddressKind> EmailAddressKinds { get; set; } = Enum.GetValues(typeof(VcardEmailAddressKind)).Cast<VcardEmailAddressKind>();
+
+        /// <summary>
+        ///    Available <see cref="VcardTelephoneNumberKind"/>s
+        /// </summary>
+        public IEnumerable<VcardTelephoneNumberKind> TelephoneNumberKinds { get; set; } = Enum.GetValues(typeof(VcardTelephoneNumberKind)).Cast<VcardTelephoneNumberKind>();
+
+        /// <summary>
+        /// Injected property to get access to <see cref="ISessionAnchor"/>
+        /// </summary>
+        private readonly ISessionAnchor SessionAnchor;
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="UserManagementPageViewModel" /> class.
+        /// </summary>
+        /// <param name="sessionAnchor">The <see cref="ISessionAnchor" /></param>
+        public UserManagementPageViewModel(ISessionAnchor sessionAnchor)
+        {
+            this.SessionAnchor = sessionAnchor;
+        }
+
+        /// <summary>
+        ///     Tries to create a new <see cref="Person" />
+        /// </summary>
+        /// <returns>A <see cref="Task" /></returns>
+        public async Task Grid_ModelSaving()
```

Comment:
please remove the word `Grid`, this is a viewmodel and the fact we are using a Grid control is not relevant. If tomorrwo we decide we don;t want to use a grid but a list or something else altogether then this method name is no longer valid.